ACE/ace/Basic_Types.h

```diff
@@ -285,9 +285,11 @@ ACE_END_VERSIONED_NAMESPACE_DECL
 // Byte swapping macros to deal with differences between little endian
 // and big endian machines.  Note that "long" here refers to 32 bit
 // quantities.
+# define ACE_SWAP_LONG_LONG(LL) ((ACE_SWAP_LONG ((LL)&0xFFFFFFFF) << 32) \
```

Comment:
We should have a test extension for this new macro

Reply:
OK, which test should I extend ? I searched in /tests but could not find any usage of the other "ACE_SWAP_x"-macros there

Reply:
Maybe Basic_Types_Test.cpp

Reply:
Done